### ISSUE TITLE
mdc-theme-text-icon-on-background does not meet the WCAG 2 AA contrast

### ISSUE THREAD

**Issue by KayvandenHeuvel** [material-components/material-components-web#4644]
### What MDC Web Version are you using?

0.41

### What browser(s) is this bug affecting?

All browsers

### What OS are you using?

Windows 10

### What are the steps to reproduce the bug?
Go to [https://material-components.github.io/material-components-web-catalog/#/component/list](https://material-components.github.io/material-components-web-catalog/#/component/list) and inspect the graphic (icon).

Check the contrast and it will show that i does not meet the AA requirements. mdc-theme-text-icon-on-background has insufficient color contrast of 2.67.

### What is the expected behavior?

 Expected contrast ratio of 3:1

### What is the actual behavior?

The color rgba(0, 0, 0, .38) has a color contrast of 2.67.

### Any other information you believe would be useful?
/

**Comment by abhiomkar** [comment]
Thanks for reporting this issue!

We're working on improving color states across components for better accessibility. I created an epic bug (#4708) to track all the things related to color contrast.
